bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/chart/defaultchartprovider/DefaultChartProvider.java

```diff
@@ -106,6 +119,16 @@ protected void activate() {
         logger.debug("Available themes for default chart provider: {}", themeNames);
     }
 
+    @Reference(cardinality=ReferenceCardinality.MANDATORY, policy=ReferencePolicy.STATIC)
```

Comment:
Mandatory-Static is the default and thus does not require to be explicitly stated here, right?

Reply:
Yes, its already implicit set, so no need to do it explicit.